packages/beacon-node/src/api/rest/base.ts

```diff
@@ -96,7 +96,7 @@ export class RestApiServer {
       if (err.validation) {
         const {instancePath, message} = err.validation[0];
         const payload: ErrorResponse = {
-          code: err.statusCode ?? 400,
```

Comment:
was this intended?

Reply:
It should probably not be part of this PR, but noticed using statusCode of error does not make sense here as it's always gonna be 400 (also to match line 103)